lib/mixpanel/types/baseTypes.ts

```diff
@@ -18,11 +18,11 @@ export interface MixpanelEventContext extends BaseEventContext {
 }
 
 export interface MixpanelBaseEventData extends BaseEventData {
-    audience: string;
+    audience?: string;
 }
 
 export interface BaseMixpanelEvent {
-    name: string;
+    name?: string;
```

Comment:
Ik zou zeggen dat de name wel altijd verplicht is. Een event moet een name hebben.

Reply:
Je had een comment op Outdated code :P